source/scss/modules/_form.scss

```diff
@@ -129,17 +129,23 @@ textarea {
   }
 }
 
-.alert {
-  display: block;
-  padding: 10px;
-  border: 3px solid $c--shady-lady;
-  color: $c--shady-lady;
-  margin-top: 5px;
-  &.hidden {
-    display: none;
-  }
-  &--red {
-    color: $c--rust;
-    border-color: $c--rust;
-  }
+.alert,
```

Comment:
All they styles look great and it's so much better when everything is consistent! Nice job!